darts/models/rnn_model.py

```diff
@@ -67,27 +72,32 @@ def __init__(self,
 
         # Defining parameters
         self.target_size = target_size
+        self.output_size = (
+            likelihood_model._num_parameters * target_size if likelihood_model is not None else target_size
+        )
         self.name = name
+        self.likelihood_model = likelihood_model
 
         # Defining the RNN module
         self.rnn = getattr(nn, name)(input_size, hidden_dim, 1, batch_first=True, dropout=dropout)
+        # TODO: dropout is not used currently since the pytorch module doesn't adds it to the last layer and we have
```

Comment:
Should we make `num_layers` a parameter?

Reply:
Yes, actually I already did that and it's merged into develop. I'm just waiting for your timeseries refactoring to be merged into develop before merging this branch with the newest version of develop. I'm hoping to avoid some annoying merge conflicts that way